ragna/_ui/api_wrapper.py

```diff
@@ -61,21 +61,28 @@ def get_chats(self):
 
         return json_data
 
-    def answer(self, chat_id, prompt):
-        json_data = (
-            self.client.post(
-                f"/chats/{chat_id}/answer", params={"prompt": prompt}, timeout=None
+    async def answer(self, chat_id, prompt):
+        async with httpx.AsyncClient(
+            base_url=self.client.base_url, headers=self.client.headers
+        ) as async_client:
```

Comment:
How much work is it to make all functions async and only use a `AsyncClient` for this whole class?

Reply:
There's one pain point : the check on the API when building the object, that leads to add async/await to the `__init__`, which isn't proper working code.
I suggest to move this check out of the init. 
I also suggest we merge #137 first to reduce conflicts on `api_wrapper.py`

Reply:
> There's one pain point : the check on the API when building the object, that leads to add async/await to the `__init__`, which isn't proper working code.

We could just use a sync client for that (or rather just use `httpx.post` since we won't reuse the client afterwards), since at this point there is no UI yet that we could render unresponsive. Plus, blocking until we actually have a proper connection to the API is probably a good thing anyway.

Reply:
We will need the sync client at other places, because there are Panel functions that donc support being async. I'm on it.